pkg/common/cns-lib/volume/manager.go

```diff
@@ -159,10 +159,12 @@ type CnsVolumeInfo struct {
 }
 
 type CnsSnapshotInfo struct {
-	SnapshotID                string
-	SourceVolumeID            string
-	SnapshotDescription       string
-	SnapshotCreationTimestamp time.Time
+	SnapshotID                          string
+	SourceVolumeID                      string
+	SnapshotDescription                 string
+	SnapshotCreationTimestamp           time.Time
+	AggregatedSnapshotCapacityInMb      int64
+	SnapshotLatestOperationCompleteTime time.Time
```

Comment:
`SnapshotCreationTimestamp` and `SnapshotLatestOperationCompleteTime` would be similar in most cases. Let's make a distinction here:

SnapshotLatestOperationCompleteTime- this the task completion time stamp
SnapshotCreationTimestamp- this is snapshot creation time stamp set by FCD

The current code seems faulty from a correctness standpoint, the snapshot creation time should either be the CNS task completion timestamp or the FCD snapshot creation time. Lets fix it here.

Set it to Task completion timestamp, if present, or else the `CnsSnapshot.CreateTime` set in CNS

We can eliminate `SnapshotCreationTimestamp` entirely, for CreateSnapshot call `SnapshotLatestOperationCompleteTime` represents the snapshot creation time.

Further, for CnsSnapshotInfo, we can simply call it `SnapshotOperationTimestamp`, in CnsVolumeInfo it makes sense to call it `SnapshotLatestOperationCompleteTime`